### ISSUE TITLE
incorrect link to docs in skeleton app.d.ts

### ISSUE THREAD

**Issue by djlauk** [sveltejs/kit#3805]
### Describe the bug

I created a skeleton-app with TypeScript support and noticed the link to the documentation inside `src/app.d.ts` is wrong.
This might be realted to today's updates to the documentation site.

I'm happy to comb through the links provided in the templates for the demo-app and skeleton and submit a PR, but I might need assistance to conform with your contribution guidelines.

### Reproduction

**Steps to reproduce**
1. Go to https://github.com/sveltejs/kit/blob/master/packages/create-svelte/templates/skeleton/src/app.d.ts#L3
2. The link in the comment says: https://kit.svelte.dev/docs#typescript
3. Open the link in this line in a new browser window

**Observed behaviour**
- You end up at the very beginning of the documentation (https://kit.svelte.dev/docs/introduction)

**Expected behaviour**
- You end up at the TypeScript specific docs (link should be https://kit.svelte.dev/docs/typescript)

### Logs

```shell
n/a
```

### System Info

```shell
n/a
```

### Severity

annoyance

### Additional Information

This is only a mild annoyance that puts a tiny dent into the otherwise very well rounded experience SvelteKit provided me with so far. I am afraid it may put people off and stop them from giving SvelteKit a try.

**Comment by djlauk** [comment]
I am nearly done preparing a PR addressing this issue. I have weeded out all the links and replaced them. Also in the generated messages, and the tests that go with them.

There is one instance left where the information I found is contradicting and I need some help on how to proceed:
- This code does a runtime check and the error links to non-existing documentation for the `target` option in the config: https://github.com/sveltejs/kit/blob/master/packages/kit/src/runtime/client/start.js#L31
- This code checks `svelte.config.js` and makes sure that the `target` option is not used: https://github.com/sveltejs/kit/blob/master/packages/kit/src/core/config/options.js#L255

I have a feeling, either the message should be changed in `start.js`, or the check should be dropped as this is no longer configurable (so, the user cannot provide nonsense).
